config/elasticsearch/mappings/feed_content.json

```diff
@@ -84,7 +84,7 @@
     "path": {
       "type": "keyword"
     },
-    "positive_reactions_count": {
+    "public_reactions_count": {
```

Comment:
I think this was an accidental replacement since you have the public_reactions_count below. Can you switch this back and put a comment here saying this field is deprecated? That way when we have a chance we can remove it.

Reply:
Fixed 👍